### PR TITLE
Add "Show splash view" to debug menu

### PR DESCRIPTION
**Pull request by takuaraki**
## Issue
- #323 

### Main
- Add "Show splash view" to debug menu ff39dca

### Other
~~I encountered leak of SplashActivity and I commit 9bfd0be to fix the leak.~~
The leak may be resolved by #329 .

## Links
N/A

### CODE REVIEW

**Comment by takuaraki** [comment]
Oh. My PR may conflict #329 ...

**Comment by konifar** [comment]
Sorry, can you merge? 🙇

**Comment by takuaraki** [comment]
I resolved conflict 😸

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -86,6 +86,11 @@
                   android:theme="@style/AppTheme.ColoredStatusBar"
                   />
 
+        <activity
+                android:name=".view.activity.SplashViewActivity"
```
Comment (konifar) [inline app/src/main/AndroidManifest.xml]: I think `SplashDemoActivity` is better name.
What do you think?

File: app/src/main/java/io/github/droidkaigi/confsched2017/MainApplication.java [hunk 2]
```diff
@@ -39,6 +40,9 @@
     NotificationStrategy notificationStrategy;
 
     @Inject
+    ShowSplashStrategy showSplashStrategy;
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/MainApplication.java]: I think it's not good to add many debug menu to debot 🤔 
No need to fix on this PR. But I might arrange the debug / developer menu later.

**Comment by konifar** [comment]
LGTM! Thanks 👍

**Pull request merged**